### DISCUSSION TITLE
Customize content manager list view

### DISCUSSION THREAD

**Discussion by Tomaszal** [strapi/strapi#7173] [Legacy Topics]
Is there any way to customize the content manager list view? More specifically I have a product collection with an internationalized component for the title. That component basically has a couple of text fields for different languages. For the API I can simply replace the find service for that collection (to get the English title for example). Is there a way to tell content manager to display that appropriately? Thanks.

**Comment by lauriejim** [comment]
Hello! I will not be able to guide you deeply in that stuff but I can tell you what logic apply.
Here is the documentation related to the customisation - https://strapi.io/documentation/v3.x/concepts/customization.html
This will let you customise the admin panel `extensions/content-manager/admin/...`
After that you will have to check in that code https://github.com/strapi/strapi/tree/master/packages/strapi-plugin-content-manager/admin/src
Where is the component you want to update to achieve your need.
The page to update/create entry is manage by the EditView container - https://github.com/strapi/strapi/tree/master/packages/strapi-plugin-content-manager/admin/src/containers/EditView

**Reply by Tomaszal** [reply]
Hello, thanks. Unfortunately I've been trying to find the exact spot where the fields for ListView table get generated yet. However I think there might be a better method to do it if what I'm thinking of is possible. Basically is it possible to create a fake field in a collection and populate it when an item is fetched?

**Reply by lauriejim** [reply]
I think this guide can help for doing that - https://github.com/strapi/strapi/pull/6433/files

**Reply by Tomaszal** [reply]
Thanks, that does partially help. However, as @derrickmehaffy points out, "since you have the field defined in the model schema, there is a column created for it". Is there any way to prevent creating a column in the database?

**Reply by derrickmehaffy** [reply]
@Tomaszal not currently no, that PR has ongoing feedback.
